MonoGame.Framework/Graphics/GraphicsDevice.DirectX.cs

```diff
@@ -567,7 +570,15 @@ internal void OnPresentationChanged()
 
 #endif
 
-        partial void PlatformValidatePresentationParameters(PresentationParameters presentationParameters)
+        partial void PlatformReset()
+        {
+#if WINDOWS
+            CorrectBackBufferSize();
+#endif
+            PlatformValidatePresentationParameters(PresentationParameters);
+        }
+
+        private void PlatformValidatePresentationParameters(PresentationParameters presentationParameters)
```

Comment:
Looks like no one else calls `PlatformValidatePresentationParameters` anymore... should I just get merged with `PlatformReset`?

Or did I miss somewhere `PlatformValidatePresentationParameters` is used?

Reply:
Yeah, it can be merged.

Reply:
Done.